dist/scrollinga.js

```diff
@@ -0,0 +1,420 @@
+	        this.removeScrollLock();
+	      }
+
+	      this.rescrolling = false;
+	    }
+
+	    /**
+	     * Rescroll on resize.
+	     */
+	  }, {
+	    key: 'onResize',
+	    value: function onResize() {
+	      this.rescroll();
+	    }
+
+	    /**
+	     * Scrolls target to the specified position in the scrollLock.
+	     */
+	  }, {
+	    key: 'rescroll',
+	    value: function rescroll() {
+	      if (this.shouldRescroll()) {
+	        this.rescrolling = true;
+	        this.target.scrollTop = this.scrollLock.at();
+	      }
+	    }
+
+	    /**
+	     * On mutation handler. Calls 'onReflow' callback on when a mutation is
+	     * detected, or an image is loaded.
+	     *
+	     * @param {Array<MutationRecord>} mutations
+	     */
+	  }, {
+	    key: 'onMutation',
+	    value: function onMutation(mutations) {
+	      this.rescroll();
+	      this.addImageListeners(mutations);
+	    }
+
+	    /**
+	     * Listen to 'load' event in new images
+	     *
+	     * @param {Array<MutationRecord>} mutations
+	     */
+	  }, {
+	    key: 'addImageListeners',
+	    value: function addImageListeners(mutations) {
+	      var _this3 = this;
+
+	      mutations.forEach(function (mutation) {
+	        var nodes = utils.toArray.call(mutation.addedNodes);
+
+	        nodes.forEach(function (node) {
+	          if (node.tagName === 'IMG') {
+	            _this3.listenToImageLoad(node);
+	          }
+
+	          if (node.getElementsByTagName) {
+	            var childImages = utils.toArray.call(node.getElementsByTagName('img'));
+
+	            childImages.forEach(function (image) {
+	              _this3.listenToImageLoad(image);
+	            });
+	          }
+	        });
+	      });
+	    }
+
+	    /**
+	     * Returns a 'load' listener for images that removes itself after being executed.
+	     *
+	     * @return {Function}
+	     */
+	  }, {
+	    key: 'getImageLoadListener',
+	    value: function getImageLoadListener() {
+	      var _this4 = this;
+
+	      var listener = function listener(event) {
+	        _this4.rescroll();
+	        event.currentTarget.removeEventListener('load', listener);
+	      };
+
+	      return listener;
+	    }
+
+	    /**
+	     * Listens to image load
+	     *
+	     */
+	  }, {
+	    key: 'listenToImageLoad',
+	    value: function listenToImageLoad(image) {
+	      image.addEventListener('load', this.getImageLoadListener());
+	    }
+
+	    /**
+	     * Removes listeners and stuff.
+	     */
+	  }, {
+	    key: 'close',
+	    value: function close() {
+	      if (this.observer) {
+	        this.observer.disconnect();
+	      }
+
+	      if (this.interval) {
+	        window.clearInterval(this.interval);
+	      }
+
+	      this.target.removeEventListener('scroll', this.onScroll);
+	      window.removeEventListener('resize', this.onResize);
+	    }
+	  }]);
+
+	  return Scrollinga;
+	})();
+
+	exports['default'] = Scrollinga;
+	module.exports = exports['default'];
+
+/***/ }
+/******/ ])
+});
+;
```

Comment:
It's just a way to pass QA. I'll remove it before publish the new scrollinga version.